tests/5.1/atomic/test_atomic_compare.c

```diff
@@ -0,0 +1,48 @@
+//===--- test_atomic_compare.c ---===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// Creates an array with random numbers, and uses atomic compare to find the max,
+// testing against non-parallel maximum.
+//
+////===----------------------------------------------------------------------===//
+#include <assert.h>
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 100
+
+int test_atomic_compare() {
+  OMPVV_INFOMSG("test_atomic_compare");
+
+  int arr[N];
+  int errors = 0;
+  int pmax, max = 0;
+
+   for(int i=0; i<N; i++){
+      arr[i] = rand()%1000;
+   }
+   for(int i = 0; i<N; i++){ // Sets max through non-parallel methods
+      if(arr[i] > max){
+         max = arr[i];
+      }
+   }
+   #pragma omp parallel for // Sets max using parallel for loop, using atomic to ensure max is correct
```

Comment:
pmax should be a shared variable

Reply:
After making this change `#pragma omp paralle for shared(pmax)` and testing with cray, clang, and gcc this should be good to go.